py-polars/tests/unit/series/test_series.py

```diff
@@ -536,6 +537,28 @@ def test_cast() -> None:
         pl.Series(["1", "2", "3", "4", "foobar"]).cast(int)
 
 
+@pytest.mark.parametrize(
```

Comment:
Can you move this test to `test_errors`?

Reply:
Done!